public/gh-reroute/404.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>AICS 3D Cell Viewer</title>
+    <style>
+      html {
+        box-sizing: border-box;
+      }
+      html,
+      body,
+      main {
+        width: 100%;
+        height: 100%;
+        margin: 0;
+        padding: 0;
+        background-color: #222222;
+        color: #bfbfbf;
+        font-family: "Open Sans", sans-serif;
+      }
+
+      #error-page-404 {
+        margin-top: 20px;
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+
+        a {
+          color: #0099ff;
+
+          &:hover,
+          &:focus-within {
+            text-decoration: underline;
+          }
+        }
+      }
+    </style>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans&display=swap" rel="stylesheet" />
+    <!-- Webpack will inject the `gh-reroute/index.tsx` script into this webpage during build. -->
```

Comment:
if webpack is truly inlining the code then that would probably be the fastest load time.  I'm wondering if it is just creating a js script link that this page is loading dynamically.

Reply:
It's currently a linked script. I'm looking into plugins to actually inject the JS into HTML...

Reply:
Okay, toyed with this a bit and I think what I have is working pretty well. I configured the script to load immediately (by disabling the `defer` attribute) and it will auto-clear the contents of the page, so you'll only ever see the "Rerouting..." text if JavaScript is disabled. This made the reroute happen much faster and feel snappier!